### ISSUE TITLE
GoogleAdvertisingId tracking inconsistency

### ISSUE THREAD

**Issue by imminent** [segmentio/analytics-android#403]
We updated to Google Play Services 8.4.0 in our last app release and we noticed that before that Segment was tracking a Google Advertising ID in the `AnalyticsContext`, but no longer is. A look into the code reveals:
```java
if (isOnClassPath("com.google.android.gms.analytics.GoogleAnalytics")) { ...
```
We don't use Google Analytics, so it's surprising that used to work, but it's also unclear the requirement of Google Analytics for retrieving the Google Advertising ID.

**Comment by f2prateek** [comment]
How else would you retrieve the advertising ID?

**Comment by imminent** [comment]
Sorry, what do you mean "how else"? 

This is code we have:
```java
AdvertisingIdClient.Info info = AdvertisingIdClient.getAdvertisingIdInfo(HotelTonightApplication.this);

if (!info.isLimitAdTrackingEnabled()) {
  adId = info.getId();
}
```

**Comment by f2prateek** [comment]
Sorry by how else I meant that the `AdvertisingIdClient` is only available when the GA dependency is present right? Which is we had that check in place. We could change it to check for `	com.google.android.gms.ads.identifier.AdvertisingIdClient`

**Comment by imminent** [comment]
No, the `AdvertisingIdClient` is not in the GA dependency, it's available with just the basic library as of 8.4 play services. We'd recommend going with your proposed change, as your current check is `false` for our app even though we use the `AdvertisingIdClient`.

**Comment by f2prateek** [comment]
Cool, updated the library. Should probably have a release out soon.
